.catalog-onboard-pipeline.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+offerings:
+  - name: deploy-arch-ibm-observability-instances
```

Comment:
remove `-instances` from the name - the offering will have instances and agents flavours

Reply:
done